client/components/Register.jsx

```diff
@@ -12,30 +12,27 @@ import { baseApiUrl as baseUrl } from '../config'
 import { labelStyle } from '../helpers'
 
 const schema = yup.object().shape({
-  email: yup
-    .string()
-    .email()
-    .required('Email is a required field'),
-  password: yup
-    .string()
-    .required('Please enter your password')
-    .matches(
-      /^.*(?=.{8,})((?=.*[!@#$%^&*()\-_=+{};:,<.>]){1})(?=.*\d)((?=.*[a-z]){1})((?=.*[A-Z]){1}).*$/,
-      'Password must contain at least 8 characters, one uppercase, one number and one special case character'
-    ),
+  email: yup.string().email().required('Email is a required field'),
+  // password: yup
+  //   .string()
+  //   .required('Please enter your password')
+  //   .matches(
+  //     /^.*(?=.{8,})((?=.*[!@#$%^&*()\-_=+{};:,<.>]){1})(?=.*\d)((?=.*[a-z]){1})((?=.*[A-Z]){1}).*$/,
+  //     'Password must contain at least 8 characters, one uppercase, one number and one special case character'
+  //   ),
```

Comment:
❕ We'll need to re-enable this logic before we can merge.

💭  Maybe there's a way we can disable password complexity check when we're not on the live version of the webapp?  Good item for the backlog...